### PR TITLE
Made required changes in code and updated readme.md file for issue-3901

### PR DESCRIPTION
**Pull request by iaditichine**
I have made required changes in the code and updated the readme.md file.

### CODE REVIEW

File: C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    while (c1--)
+    {
+        cout << 1 << " "; //printing 1 for c1 times
+    }
+    cout << endl;
+}
+int main()
+{
+
+    ios::sync_with_stdio(0);
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp:53]: Don't use CP references

File: C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+        c1++;
+    }
+    while (c0--)
+    {
+        cout << 0 << " "; //printing 0 for c0 times
+    }
+    while (c1--)
+    {
+        cout << 1 << " "; //printing 1 for c1 times
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp:46]: Use comments above the line of code not beside it

File: C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+Output-1: 0 0 0 1 1
+
+*/
+//CODE
+#include <iostream>
+using namespace std;
+//Function to sort an array of 0's and 1's
+void sort()
+{
+	//Input n= Size of array
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp:17]: Remove redundant comments

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
We don't accept PRs without any corresponding Issue. It is difficult for us to find your issue in tons of issues available. We can't review and merged incomplete PR's.  Closing this PR as of now. Open the new PR if you want this to be reviewed and add the proper details.

**Pull request closed without merging**